src/build.py

```diff
@@ -388,9 +389,13 @@ def pretty(fmt):
       return proc.check_output(
           ['git', 'log', '-n1', '--pretty=format:%s' % fmt],
           cwd=self.src_dir).strip()
-    remote_name = 'remote.%s.url' % WATERFALL_REMOTE
-    remote = proc.check_output(['git', 'config', '--get', remote_name],
-                               cwd=self.src_dir).strip()
+    try:
+      remote = GitRemoteUrl(self.src_dir, WATERFALL_REMOTE)
+    except proc.CalledProcessError:
+      # Not all checkouts have the '_waterfall' remote (e.g. the waterfall
```

Comment:
The waterfall syncs itself? I thought its custom_sync was set to NoSync meaning it didn't sync?